hw-model/src/lib.rs

```diff
@@ -174,18 +177,33 @@ impl<'a> Default for InitParams<'a> {
             etrng_responses,
             trng_mode: Default::default(),
             wdt_timeout_cycles: EXPECTED_CALIPTRA_BOOT_TIME_IN_CYCLES,
+            soc_apb_pauser: 0x1,
         }
     }
 }
 
-#[derive(Default)]
 pub struct BootParams<'a> {
     pub init_params: InitParams<'a>,
     pub fuses: Fuses,
     pub fw_image: Option<&'a [u8]>,
     pub initial_dbg_manuf_service_reg: u32,
     pub initial_repcnt_thresh_reg: Option<CptraItrngEntropyConfig1WriteVal>,
     pub initial_adaptp_thresh_reg: Option<CptraItrngEntropyConfig0WriteVal>,
+    pub valid_pauser: u32,
+}
+
+impl<'a> Default for BootParams<'a> {
+    fn default() -> Self {
+        Self {
+            init_params: Default::default(),
+            fuses: Default::default(),
+            fw_image: Default::default(),
+            initial_dbg_manuf_service_reg: Default::default(),
+            initial_repcnt_thresh_reg: Default::default(),
+            initial_adaptp_thresh_reg: Default::default(),
+            valid_pauser: 0x1,
```

Comment:
Why does it need to be in BootParams and InitParams?

Reply:
One is for configuring the VALID_PAUSER register, the other is for which PAUSER will be used for bus accesses from "SoC". I'm not completely against having them be one thing, but I figured the control to individually control them would be wanted.

Additionally I'd like to add a test case to confirm accesses are dropped when there is a mismatch (which would require the individual control), but that's not supported in the emulator and I don't know if I can classify a test as invalid in the emulator environment.